src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -445,6 +452,27 @@ private static boolean hasSpeciesAttribute(GKInstance physicalEntity) {
         return physicalEntity.getSchemClass().isValidAttribute(ReactomeJavaConstants.species);
     }
 
+    /**
+     * Helper method that finds all unique DbIds in the 'species' and 'relatedSpecies' attributes of the instance.
+     * @param inst - GKInstance, an Event or PhysicalEntity.
+     * @return - Set<Long>, DbIds from species/relatedSpecies attributes.
+     * @throws Exception, thrown by MySQLAdaptor.
+     */
+    public static Set<Long> getSpeciesAndRelatedSpeciesDbIds(GKInstance inst) throws Exception {
+        Set<Long> speciesDbIds = new HashSet<>();
+        if (inst.getSchemClass().isValidAttribute(ReactomeJavaConstants.species)) {
+            for (GKInstance speciesInst : (Collection<GKInstance>) inst.getAttributeValuesList(ReactomeJavaConstants.species)) {
+                speciesDbIds.add(speciesInst.getDBID());
+            }
+        }
+        if (inst.getSchemClass().isValidAttribute(ReactomeJavaConstants.relatedSpecies)) {
+            for (GKInstance relatedSpeciesInst : (Collection<GKInstance>) inst.getAttributeValuesList(ReactomeJavaConstants.relatedSpecies)) {
+                speciesDbIds.add(relatedSpeciesInst.getDBID());
+            }
+        }
```

Comment:
Not a big deal, but you could combine these in a for-loop for the species and relatedSpecies attributes:
```
List<String> speciesAttributes = Arrays.asList(ReactomeJavaConstants.species, ReactomeJavaConstants.relatedSpecies);
for (String speciesAttribute: speciesAttributes) {
   if(inst.getSchemClass().isValidAttribute(speciesAttribute)) {
       for (GKInstance speciesInst : (Collection<GKInstance>) inst.getAttributeValuesList(speciesAttribute)) {
           speciesDbIds.add(speciesInst.getDbId())
       }
   }
}
```